html/janus.js

```diff
@@ -1752,11 +1726,11 @@ function Janus(gatewayCallbacks) {
 					pluginHandle.iceState(config.pc.iceConnectionState);
 			};
 			config.pc.onicecandidate = function(event) {
-				if (event.candidate == null ||
+				if (!event.candidate ||
 						(Janus.webRTCAdapter.browserDetails.browser === 'edge' && event.candidate.candidate.indexOf('endOfCandidates') > 0)) {
 					Janus.log("End of candidates.");
 					config.iceDone = true;
-					if(config.trickle === true) {
+					if(config.trickle) {
```

Comment:
This should actually remain a `true` check, the `trickle` property must be a boolean.